src/sidebar/components/test/MarkdownView-test.js

```diff
@@ -8,14 +12,31 @@ describe('MarkdownView', () => {
   let fakeRenderMathAndMarkdown;
   let fakeReplaceLinksWithEmbeds;
   let fakeRenderMentionTags;
+  let fakeClearTimeout;
 
-  const markdownSelector = '[data-testid="markdown-text"]';
+  function createComponent(props = {}) {
+    return mount(
+      <MarkdownView
+        markdown=""
+        {...props}
+        setTimeout_={callback => setTimeout(callback, 1)}
```

Comment:
You should be able to use `setTimeout(callback, 0)` here and it will be even quicker (next event loop turn instead of at least one ms).